Assets/HoloToolkit/Utilities/Scripts/HeadsetAdjustment.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using HoloToolkit.Unity.InputModule;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class HeadsetAdjustment : MonoBehaviour, IInputClickHandler, ISpeechHandler {
+
+    [Tooltip("The name of the scene to load when the user is ready. If left empty, the next scene is loaded as specified in the 'Scenes in Build')")]
+    public string NextSceneName;
+
+    public void OnInputClicked(InputClickedEventData eventData)
+    {
+        Debug.Log("Clicked");
+        GotoNextScene();
+    }
+
+    private void GotoNextScene()
+    {
+        if (!string.IsNullOrEmpty(NextSceneName))
+        {
+            SceneManager.LoadScene(NextSceneName);
+        }
+        else
+        {
+            var sceneIndex = SceneManager.GetActiveScene().buildIndex;
+            SceneManager.LoadScene(sceneIndex + 1);
```

Comment:
It throws an exception. I decided not to be defensive about this, since the exception thrown by Unity itself is clear enough, and the property's description clearly points to what went wrong. Repeating / rephrasing the exception wouldn't have added much.